src/components/daily-auths-report.tsx

```diff
@@ -13,6 +13,7 @@ import Accordion from "./accordion";
 import useResizeListener from "../hooks/resize-listener";
 import { ProcessedResult, loadData } from "../models/daily-auths-report-data";
 import { formatSIDropTrailingZeroes, formatWithCommas, yearMonthDayFormat } from "../formats";
+import kebabCase from "lodash.kebabcase";
```

Comment:
I have some recollection that these function-specific packages were deprecated. You could get the best of both worlds using `lodash-es` which is tree-shakable, though it is pulling down the full library during `npm install`. 

Alternatively, we implement a basic version of it inline in the IdP:

https://github.com/18F/identity-idp/blob/c91a0cf9e82e64e3206152b9d6860297f1f907aa/app/javascript/packs/form-validation.js#L15-L17

Reply:
yessss thank you I thought we'd done something like that but forgot to check the IDP codebase, updated in 0421dcc1a457de2384da393da9800cc0664b2d32